manala.nginx/defaults/main.yml

```diff
@@ -14,7 +14,7 @@ manala_nginx_config:          []
 manala_nginx_configs_exclusive: false
 manala_nginx_configs_dir:       /etc/nginx/conf.d
 manala_nginx_configs_template:  ~
-manala_nginx_configs:           []
+manala_nginx_configs:          []
```

Comment:
Woops, i guess something has escaped your shrewdness :)

Reply:
Damn... Hope you'll sleep anyway until monday :S